### PR TITLE
EPMRPP-78039 || Fixed links for "Azure SAML" and "Active Directory" that lead to blank pages

### PR DESCRIPTION
**Pull request by Bam6ycha**
Jira [EPMRPP-78039]. Landing page. ReportPortal configuration. Integration section. Links for "Azure SAML" and "Active Directory" lead to blank pages

### CODE REVIEW

File: src/md/src/GettingStarted/Integrations.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ Users can reinforce ReportPortal with adding additional integrtions with:
 * [Sauce Labs](https://reportportal.io/docs/Sauce-Labs-integration)
 * [E-mail server](https://reportportal.io/docs/E-mail-server-integration)
 * [LDAP](https://reportportal.io/docs/LDAP-Auth-integration)
-* [Active Directory](https://reportportal.io/docs/Active-Directory-Auth-integration)
```
Comment (chivekrodis) [inline src/md/src/GettingStarted/Integrations.md:14]: Maybe better add `-integration` postfix in the `Plugins.md` file to be more consistent

File: src/md/src/GettingStarted/Plugins.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ Details on how to configure a specific plugin please find in the [Integration se
 * [integration with LDAP](https://reportportal.io/docs/LDAP-Auth-integration)
 * [integration with Active directory](https://reportportal.io/docs/Active-Directory-Auth)
 * [integration with Okta SAML](https://reportportal.io/docs/Okta-SAML-integration)
-* [integration with Azure SAML](https://reportportal.io/docs/Azure-SAML-integration)
+* [integration with Azure SAML](https://reportportal.io/docs/Azure-SAML-Integration)
```
Comment (chivekrodis) [inline src/md/src/GettingStarted/Plugins.md:105]: Please use `-integration` in lower case to be more consistent across all items here and every places where `https://reportportal.io/docs/Azure-SAML-Integration` used

**Review by chivekrodis — CHANGES_REQUESTED** [review]
Please rename PR according to our rules. You can fined info [here](https://github.com/reportportal/service-ui/blob/master/docs/13-pullRequests.md)

@AmsterGet do we have plans move to another parser or at least make urls are case insensitive?

**Comment by AmsterGet** [comment]
> Please rename PR according to our rules. You can find info [here](https://github.com/reportportal/service-ui/blob/master/docs/13-pullRequests.md)
> 
> @AmsterGet do we have plans move to another parser or at least make urls are case insensitive?

We will implement the new documentation using modern and more convenient approach, so I think this issue will be unactual after that.

**Comment by chivekrodis** [comment]
> Jira [EPMRPP-78039]. Landing page. ReportPortal configuration. Integration section. Links for "Azure SAML" and "Active Directory" lead to blank pages

EPMRPP-78029 || short description like fix ad, azure saml integration links will be enough description it's up to you but please use convention `[project ticket] || description`

**Pull request closed without merging**